save-backend/src/test/kotlin/com/saveourtool/save/backend/controllers/RunExecutionControllerTest.kt

```diff
@@ -49,6 +51,7 @@ class RunExecutionControllerTest(
     @Autowired private lateinit var testExecutionRepository: TestExecutionRepository
     @Autowired private lateinit var lnkExecutionFileRepository: LnkExecutionFileRepository
     @Autowired private lateinit var lnkExecutionTestSuiteRepository: LnkExecutionTestSuiteRepository
+    @MockBean private lateinit var lnkVulnerabilityUserRepository: LnkVulnerabilityUserRepository
```

Comment:
if you don't use this property, please move this bean to annotation section only:
```
@MockBeans(
    MockBean(LnkVulnerabilityUserRepository::class),
)
```